plugins/modules/cargo.py

```diff
@@ -58,15 +58,86 @@
     type: bool
     default: false
     version_added: 7.5.0
+  force:
+    description:
+      - Forcefully install package(s).
+      - Ignores many checks, and updates/installs forcefully.
+    type: bool
+    default: false
+    required: false
+    version_added: 8.2.0
   state:
     description:
       - The state of the Rust package.
     required: false
     type: str
     default: present
     choices: [ "present", "absent", "latest" ]
+  features:
+    description:
+      - List of features to enable for the package.
+      - Don't use with multiple names
+    type: list
+    elements: str
+    required: false
+    default: []
+    version_added: 8.2.0
+  all_features:
+    description:
+      - Enable all features for package(s).
+    type: bool
+    default: false
+    required: false
+    version_added: 8.2.0
+  directory:
+    description:
+      - Path to install the package(s) from.
+    type: path
+    required: false
+    version_added: 8.2.0
+  registry:
+    description:
+      - Registry to install the package(s) from.
+    type: str
+    required: false
+    version_added: 8.2.0
+  git:
+    description:
+      - Git URL to install the package(s) from.
+      - Can optionally either be used with C(branch), C(tag) or C(rev).
+    type: str
+    required: false
+    version_added: 8.2.0
+  branch:
+    description:
+      - Git branch to install the package(s) from.
+      - C(git) option must be specified.
+    type: str
+    required: false
+    version_added: 8.2.0
+  tag:
+    description:
+      - Git tag to install the package(s) from.
+      - C(git) option must be specified.
+    type: str
+    required: false
+    version_added: 8.2.0
+  rev:
+    description:
+      - Git commit to install the package(s) from.
+      - C(git) option must be specified.
+    type: str
+    required: false
+    version_added: 8.2.0
+  debug:
+    description:
+      - Specify whether to install package(s) in debug mode.
+    type: bool
+    default: false
+    required: false
+    version_added: 8.2.0
 requirements:
-    - cargo installed
+    - cargo  and git installed
```

Comment:
Adding a requirement is only allowed if it's only needed for new features (otherwise it's a breaking change). In that case it has to be explicitly documented when exactly the new requirement is needed.

Also a TOML library is needed for some features, which isn't mentioned here at all.